### PR TITLE
Removing usage of lodash _.has

### PR DESCRIPTION
**Pull request by pnkapadia64**
#302 

Replacing instances of lodash `_.has` with native `in`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: __tests__/buildFiles.test.js [hunk 1]
```diff
@@ -93,13 +93,13 @@ describe('buildFiles', () => {
 
   it('should throw if build path doesn\'t have a trailing slash', () => {
     expect(
-      buildFiles.bind(null, dictionary, platformWithBadBuildPath),
```
Comment (dbanksdesign) [inline __tests__/buildFiles.test.js]: Why did these lines change?

File: __tests__/buildFiles.test.js [hunk 2]
```diff
@@ -93,13 +93,13 @@ describe('buildFiles', () => {
...
+      buildFiles.bind(null, dictionary, platformWithBadBuildPath)
```
Comment (chazzmoney) [inline __tests__/buildFiles.test.js]: lets undo this format change for now, we can put in a separate PR if necessary

File: __tests__/buildFiles.test.js [hunk 3]
```diff
@@ -93,13 +93,13 @@ describe('buildFiles', () => {
...
     ).toThrow('Build path must end in a trailing slash or you will get weird file names.');
   });
 
   it('should throw if missing a format', () => {
     expect(
-      buildFiles.bind(null, dictionary, platformWithoutFormatter),
+      buildFiles.bind(null, dictionary, platformWithoutFormatter)
```
Comment (chazzmoney) [inline __tests__/buildFiles.test.js]: lets undo this format change for now, we can put in a separate PR if necessary

File: __tests__/buildPlatform.test.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ describe('buildPlatform', () => {
 
   it('should throw if passed a platform that doesn\'t exist', () => {
     expect(
-      StyleDictionaryExtended.buildPlatform.bind(test, 'foobar'),
+      StyleDictionaryExtended.buildPlatform.bind(test, 'foobar')
```
Comment (chazzmoney) [inline __tests__/buildPlatform.test.js]: lets undo this format change for now, we can put in a separate PR if necessary

File: __tests__/buildPlatform.test.js [hunk 5]
```diff
@@ -171,7 +171,7 @@ Unknown transformGroup "bar" found in platform "foo":
 `;
 
     expect(
-      StyleDictionaryExtended.buildPlatform.bind(StyleDictionaryExtended, 'foo'),
+      StyleDictionaryExtended.buildPlatform.bind(StyleDictionaryExtended, 'foo')
```
Comment (chazzmoney) [inline __tests__/buildPlatform.test.js]: lets undo this format change for now, we can put in a separate PR if necessary

File: lib/buildPlatform.js [hunk 6]
```diff
@@ -42,7 +40,7 @@ var flattenProperties = require('./utils/flattenProperties'),
 function buildPlatform(platform) {
   console.log('\n' + platform);
 
-  if (!this.options || !_.has(this.options.platforms, platform)) {
+  if (!this.options || !(platform in (this.options.platforms || {}))) {
```
Comment (dbanksdesign) [inline lib/buildPlatform.js]: TIL the 'in' operator in JS!

File: __tests__/buildFiles.test.js (hunk 2 above)
Comment (pnkapadia64) [reply __tests__/buildFiles.test.js]: The pre-commit hook (lint script) fails without this

Comment (dbanksdesign) [reply __tests__/buildFiles.test.js]: That is weird, it shouldn't. Can you be sure you are using our version of eslint and not a globally installed version? Double check you have `eslint` in node_modules/.bin and make sure you run `npm ci` to install exact dependency versions. You also might have an issue with your $PATH, you can run `npm run lint --verbose` to make sure local executables are before any global paths.

**Comment by pnkapadia64** [comment]
Hey @dbanksdesign @chazzmoney , I've removed the unnecessary changes in test files that came up before as lint errors. Thank you.

**Comment by chazzmoney** [comment]
:shipit:
